pkg/build/builder/source.go

```diff
@@ -392,12 +393,18 @@ func extractSourceFromImage(ctx context.Context, dockerClient DockerClient, stor
 		}
 	}
 
+	defaultContainerConfig, err := cconfig.Default()
+	if err != nil {
+		return err
+	}
+	capabilities := defaultContainerConfig.Capabilities("", nil, dropCapabilities())
+
 	builderOptions := buildah.BuilderOptions{
-		FromImage:        image,
-		PullPolicy:       pullPolicy,
-		ReportWriter:     os.Stdout,
-		SystemContext:    &systemContext,
-		DropCapabilities: dropCapabilities(),
+		FromImage:     image,
+		PullPolicy:    pullPolicy,
+		ReportWriter:  os.Stdout,
+		SystemContext: &systemContext,
+		Capabilities:  capabilities,
```

Comment:
I thought you wanted to revert this change in Buildah?

Reply:
We reverted the change in the `imagebuildah` package, but the main `buildah` package still included them.  If we’re also going to change those back, then it’ll resolve the `transientMounts` question above, else we’ll have to adapt.

Reply:
> else we’ll have to adapt.

@nalind what does this entail?

Reply:
If the API changes aren't being reverted, then we'll still need the commit that adjusts us to the new API.  Otherwise we can drop it and keep only the one that updates our dependencies list and the vendored copies of the deps.

Reply:
@nalind I guess for me it comes down to whether or not this new API will be permanent, or if it will subsequently get reveted in buildah v1.15. If the former we can keep it as is.

Reply:
@rhatdan what are your intentions there?

Reply:
I thought you wanted me to revert these changes from buildah to work with the previous API.